src/screens/Logout.jsx

```diff
@@ -1,11 +1,13 @@
 import React, { useEffect } from "react";
 import { logout } from "../utils/request";
 
-const Logout = () => {
+const Logout = ({ deleteUserSession }) => {
+  
   useEffect(() => {
     const onMount = async () => {
       localStorage.setItem("role", "");
       localStorage.setItem("token", "");
+      deleteUserSession();
```

Comment:
Call setUser(false) from the context here, instead of passing deleteUserSession from parent.